docs/node-groups.md

```diff
@@ -65,6 +65,267 @@ The below example demonstrates advanced configuration options for a managed node
+
+      k8s_taints = []
+
+      k8s_labels = {
+        Environment = "preprod"
+        Zone        = "dev"
+        WorkerType  = "ON_DEMAND"
+      }
+      additional_tags = {
+        ExtraTag    = "m5x-on-demand"
+        Name        = "m5x-on-demand"
+        subnet_type = "private"
+      }
+      create_worker_security_group = false
+    },
+    
+    #---------------------------------------------------------#
+    # ARM instance type Worker Group
+    #---------------------------------------------------------#
+    arm = {
+      # 1> Node Group configuration - Part1
+      node_group_name        = "arm-mg5"         # Max 40 characters for node group name
+      create_launch_template = true              # false will use the default launch template
+      launch_template_os     = "amazonlinux2eks" # amazonlinux2eks or bottlerocket
+      public_ip              = false             # Use this to enable public IP for EC2 instances; only for public subnets used in launch templates ;
+      pre_userdata           = <<-EOT
+            yum install -y amazon-ssm-agent
+            systemctl enable amazon-ssm-agent && systemctl start amazon-ssm-agent"
+        EOT
+      # 2> Node Group scaling configuration
+      desired_size    = 2
+      max_size        = 2
+      min_size        = 2
+      max_unavailable = 1 # or percentage = 20
+
+      # 3> Node Group compute configuration
+      ami_type       = "AL2_ARM_64" # AL2_x86_64, AL2_x86_64_GPU, AL2_ARM_64, CUSTOM, BOTTLEROCKET_ARM_64, BOTTLEROCKET_x86_64
+      capacity_type  = "ON_DEMAND"  # ON_DEMAND or SPOT
+      instance_types = ["m5.large"] # List of instances used only for SPOT type
+      disk_size      = 50
+
+      # 4> Node Group network configuration
+      subnet_ids = [] # Defaults to private subnet-ids used by EKS Controle plane. Define your private/public subnets list with comma separated subnet_ids  = ['subnet1','subnet2','subnet3']
+
+      k8s_taints = []
+
+      k8s_labels = {
+        Environment = "preprod"
+        Zone        = "dev"
+        WorkerType  = "ON_DEMAND"
+      }
+      additional_tags = {
+        ExtraTag    = "m5x-on-demand"
+        Name        = "m5x-on-demand"
+        subnet_type = "private"
+      }
+      create_worker_security_group = false
+    },
+
+    #---------------------------------------------------------#
+    # Bottlerocket ARM instance type Worker Group
+    #---------------------------------------------------------#
+    bottlerocket_arm = {
+      # 1> Node Group configuration - Part1
+      node_group_name        = "btl-arm"      # Max 40 characters for node group name
+      create_launch_template = true           # false will use the default launch template
+      launch_template_os     = "bottlerocket" # amazonlinux2eks or bottlerocket
+      public_ip              = false          # Use this to enable public IP for EC2 instances; only for public subnets used in launch templates ;
+      pre_userdata           = <<-EOT
```

Comment:
this user data is incorrect for bottlerocket; bottlerocket uses TOML and doesn't have `yum`, `systemctl`, etc.

Reply:
Correct. We can remove the `pre_userdata`  for the docs now

Reply:
Thank you both, this has been addressed, removed pre_userdata from bottlerocket docs